docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -146,6 +197,7 @@ You are now able to connect to the database as the FHIRSERVER user using only th
 
 The [Db2 driver](https://repo1.maven.org/maven2/com/ibm/db2/jcc/11.5.0.0/jcc-11.5.0.0.jar) is able to execute a connectivity test to check the configuration of the combo of API-key/Service-Id/Db2-User-Id.
 
+#### Test IAM Access
```

Comment:
If we keep both Db2 and IAM options above, then I think we should give them in the same order here.
Db2 first, then IAM.

Reply:
Yep, I've flipped them